modules/flowable-cmmn-engine/src/main/java/org/flowable/cmmn/engine/test/impl/CmmnTestRunner.java

```diff
@@ -207,16 +207,16 @@ protected void assertDatabaseEmpty(FrameworkMethod method) {
             cmmnEngineConfiguration,
             TABLENAMES_EXCLUDED_FROM_DB_CLEAN_CHECK,
             true,
-            new Command<Void>() {
-
-                @Override
-                public Void execute(CommandContext commandContext) {
-                    SchemaManager schemaManager = CommandContextUtil.getCmmnEngineConfiguration(commandContext).getSchemaManager();
-                    schemaManager.schemaDrop();
-                    schemaManager.schemaCreate();
-                    return null;
+                new Command<>() {
```

Comment:
This looks like the formatting was changed as well. Is that mandatory?

Reply:
reverted to original foirmatting